### PR TITLE
Add some detail to networking release notes for 1.1

### PR DESCRIPTION
**Pull request by louiscryan**
Fleshed out some of the networking release notes

### CODE REVIEW

**Comment by geeknoid** [comment]
Looks good, just a few CI items to take care of.

Load balancer is two words.
DNS-style needs to be added to the .spelling dictionary file.

**Comment by louiscryan** [comment]
Thanks. Pinging Shriram for review on some details.

File: content/about/notes/1.1/index.md [hunk 1]
```diff
@@ -49,23 +49,34 @@ Istio 1.1.
...
+  reduces the amount of configuration computed and transmitted to
+  the proxy, improving startup time, resource consumption and control-plane scalability. 
+  For large deployments, we recommend adding a sidecar resource per namespace.
+
+- **Restrict Visibility of Services**. Added the new `exportTo` feature which allows
+  service owners to control which namespaces can reference their services. This feature is 
+  added to `ServiceEntry`, `VirtualService` and is also supported on a Kubernetes Service via the
+  `networking.istio.io/exportTo` annotation. 
+  
+- **Namespace-Scoped Service References**. When referring to a service (host) we use DNS-style name matching
```
Comment (rshriram) [inline content/about/notes/1.1/index.md]: per offline discussion, I wouldn't portray this concept as an alt way of referring to services just yet, until we resolve all the issues associated with it. For now, I would simply say that gateways can refer to specific virtual services in a namespace using the namespace/host syntax.. Similarly, virtual services can refer to specific gateways in a namespace using the namespace/gatewayName syntax..

The sidecar's format is not a change over existing. It is completely new and syntax/semantics are conveyed in the docs..

Comment (louiscryan) [reply content/about/notes/1.1/index.md]: Done. PTAL

**Comment by geeknoid** [comment]
Don't forget these:

./content/about/notes/1.1/index.md:53: MD009 Trailing spaces
./content/about/notes/1.1/index.md:54: MD009 Trailing spaces
./content/about/notes/1.1/index.md:56: MD009 Trailing spaces
./content/about/notes/1.1/index.md:61: MD009 Trailing spaces
./content/about/notes/1.1/index.md:63: MD009 Trailing spaces
./content/about/notes/1.1/index.md:64: MD009 Trailing spaces
./content/about/notes/1.1/index.md:67: MD009 Trailing spaces
./content/about/notes/1.1/index.md:68: MD009 Trailing spaces
./content/about/notes/1.1/index.md:69: MD009 Trailing spaces
./content/about/notes/1.1/index.md:77: MD009 Trailing spaces

File: content/about/notes/1.1/index.md [hunk 2]
```diff
@@ -50,31 +50,31 @@ Istio 1.1.
...
+  `networking.istio.io/exportTo` annotation.
+
 - **Namespace Scoping**. When referring to a `VirtualService` in a Gateway we use DNS-style name matching
   in our configuration model. This can be ambiguous when more than one namespace defines a virtual service
-  for the same host name. To resolve ambiguity it is now possible to explicitly scope these references 
-  by namespace using a syntax of the form **`[{namespace-name}]/{hostname-match}`** in the `hosts` field. 
-  The equivalent capability is also available in `Sidecar` for egress.  
+  for the same host name. To resolve ambiguity it is now possible to explicitly scope these references
+  by namespace using a syntax of the form **`[{namespace-name}]/{hostname-match}`** in the `hosts` field.
+  The equivalent capability is also available in `Sidecar` for egress.
```
Comment (linsun) [inline content/about/notes/1.1/index.md]: Is this required now to use namespace/hostname for a VS to refer to a gateway in a different namespace?   I assume only needed when there is ambiguity.

**Comment by linsun** [comment]
thank you @louiscryan for adding the details - it was helpful!

File: .spelling [hunk 3]
```diff
@@ -136,6 +136,7 @@ Devirtualization
 devops
 discuss.istio.io
 distro
+DNS-style
```
Comment (sdake) [inline .spelling]: i think you can remove this line.

File: content/about/notes/1.1/index.md [hunk 4]
```diff
@@ -49,15 +49,24 @@ Istio 1.1.
...
+  the proxy, improving startup time, resource consumption and control-plane scalability.
+  For large deployments, we recommend adding a sidecar resource per namespace. Controls are also
+  provided for ports, protocols and traffic capture for advanced use-cases.
+
+- **Restrict Visibility of Services**. Added the new `exportTo` feature which allows
+  service owners to control which namespaces can reference their services. This feature is
+  added to `ServiceEntry`, `VirtualService` and is also supported on a Kubernetes Service via the
+  `networking.istio.io/exportTo` annotation.
+
+- **Namespace Scoping**. When referring to a `VirtualService` in a Gateway we use DNS-style name matching
```
Comment (sdake) [inline content/about/notes/1.1/index.md]: and just change "DNS-style" to "DNS based"

File: content/about/notes/1.1/index.md [hunk 5]
```diff
@@ -49,15 +49,24 @@ Istio 1.1.
...
-- **New `Sidecar` Resource**. Added support to limit the set of services
-  visible to sidecar proxies in a given namespace using the `Sidecar` resource.
-  This limit reduces the amount of configuration computed and transmitted to
-  the proxy. On large clusters, we recommend adding a sidecar resource per
-  namespace.
-
-- **Restrict Visibility of Networking Resources**. Added the new `exportTo`
-  field to all networking resources which lets you control the visibility of
-  individual resources to specific namespaces.
+- **New [`Sidecar`](/docs/concepts/traffic-management/#sidecars) Resource**. Enables more fine-grained
```
Comment (sdake) [inline content/about/notes/1.1/index.md]: links in the subject lead-ins feel a little one-off and strange.  Can you try `**New `Sidecar` Resource**.  Added support via the [sidecar resource](/docs/concepts/traffic-management/#sidecars).  This feature enables...`

**Comment by sdake** [comment]
@louiscryan if you make the suggested changes, I can merge.  As is I don't have ACLs to merge spelling changes and we are striving to keep the spelling file minimized.

Cheers
-steve

File: content/about/notes/1.1/index.md (hunk 5 above)
Comment (sdake) [reply content/about/notes/1.1/index.md]: also links don't need backticks.

File: .spelling (hunk 3 above)
Comment (louiscryan) [reply .spelling]: Done

File: content/about/notes/1.1/index.md (hunk 5 above)
Comment (louiscryan) [reply content/about/notes/1.1/index.md]: Done

**Comment by louiscryan** [comment]
@sdake resolved your comments I think

File: content/about/notes/1.1/index.md [hunk 6]
```diff
@@ -49,10 +49,10 @@ Istio 1.1.
 
 ## Traffic management
 
-- **New [`Sidecar`](/docs/concepts/traffic-management/#sidecars) Resource**. Enables more fine-grained
-  control over the behavior of the sidecar proxies attached to workloads within a namespace. In
-  particular adds support to limit the set of services a sidecar will send traffic to. This
-  reduces the amount of configuration computed and transmitted to
+- **New `Sidecar` Resoruce**. The new [sidecar](/docs/concepts/traffic-management/#sidecars) resource
```
Comment (sdake) [inline content/about/notes/1.1/index.md]: I'm not sure if I mispelled this, but should be `Resource` rather than `Resoruce`.

**Comment by louiscryan** [comment]
I blame the typo on my evil thumb....

**Pull request merged**